src/server/qgshttprequesthandler.cpp

```diff
@@ -505,6 +511,8 @@ bool QgsHttpRequestHandler::startGetFeatureResponse( QByteArray* ba, const QStri
     format = "text/xml";
 
   setInfoFormat( format );
+  startStreamingResponse();
```

Comment:
Hi @AsgerPetersen you didn't need to create startStreamingResponse.
To fix it, just add **sendHeaders();** just after *line 507* **setInfoFormat( format );** and it's worked.

Can I close your Pull-Request and push my fix ?